docs/source/en/api/schedulers/overview.md

```diff
@@ -12,81 +12,49 @@ specific language governing permissions and limitations under the License.
 
 # Schedulers
 
-Diffusers contains multiple pre-built schedule functions for the diffusion process.
-
-## What is a scheduler?
-
-The schedule functions, denoted *Schedulers* in the library take in the output of a trained model, a sample which the diffusion process is iterating on, and a timestep to return a denoised sample. That's why schedulers may also be called *Samplers* in other diffusion models implementations.
-
-- Schedulers define the methodology for iteratively adding noise to an image or for updating a sample based on model outputs.
-    - adding noise in different manners represent the algorithmic processes to train a diffusion model by adding noise to images.
-    - for inference, the scheduler defines how to update a sample based on an output from a pretrained model.
-- Schedulers are often defined by a *noise schedule* and an *update rule* to solve the differential equation solution.
-
-### Discrete versus continuous schedulers
-
-All schedulers take in a timestep to predict the updated version of the sample being diffused.
-The timesteps dictate where in the diffusion process the step is, where data is generated by iterating forward in time and inference is executed by propagating backwards through timesteps.
-Different algorithms use timesteps that can be discrete (accepting `int` inputs), such as the [`DDPMScheduler`] or [`PNDMScheduler`], or continuous (accepting `float` inputs), such as the score-based schedulers [`ScoreSdeVeScheduler`] or [`ScoreSdeVpScheduler`].
```

Comment:
I'd be in favor of keeping this information. Any reason not to?

Reply:
Most of the information is still there, I just made it less wordy and more concise. The only information I removed is how schedulers are designed (this can be found in the Philosophy).

Reply:
From the commits, it felt like the sections (that should be present like this one) are entirely gone.

Reply:
Hmm that's strange, I can still see all the info there:

![Screenshot 2023-08-02 at 10 26 39 AM](https://github.com/huggingface/diffusers/assets/59462357/1d080f23-bbe4-43b2-9c09-904c42eda1c8)